documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: documentation
+menu: check
+subtitle: 'Tutorial: Using Check with SQL Server'
+---
+# Tutorial: Using Flyway Check with SQL Server
+
+{% include enterprise.html %}
+
+## Introduction
+
+This is an enteprise feature, but the beta version is available to Teams users.
```

Comment:
Spelling - Enterprise
...is available to Teams users ... during the beta period ?